pages/categories/downstream.md

```diff
@@ -10,12 +10,97 @@ Downstream emissions (Category D) are the emissions produced by using an organis
+This considers the emissions generated from the electricity consumption of devices such as desktops, laptops, tablets, and mobile phones that utilise the products or services provided by the organisation. It is important to account for the differences in energy efficiency across various devices accessing these products and services. Typically, a smartphone uses less energy than a laptop, and a laptop uses less energy than a desktop. Understanding the energy use of devices can then be used to estimate carbon emissions by considering the source of the energy used to power or charge that device. Finally, end-user devices should have their embodied carbon estimated. This should take into account the product's life span to consider the entire [Product Life Cycle](/glossary#product-life-cycle).
+
+{% include linkedHeading.html heading="End-User Device Energy Consumption" level=3 %}
+Power range and average power of devices, data compiled from Dimpact [^dimpact], Scope3 [^scope3] and Fershad Irani [^fershad] and our own device energy use research:
+
+| Device             | Power Range (W)    | Average Power (W)  |
+| ------------------ | ------------------ | ------------------ |
+| Desktop            | 72 - 100           | 72.3               |
+| Laptop             | 15 - 30            | 17.1               |
+| Tablet             | 3 - 5.5            | 3                  |
+| Smartphone         | 0.77 - 2           | 1                  |
+| Monitor            | 17 - 30            | 30                 |
+| Television         | 40 - 120           | 74                 |
+
+Based on Scope3 and Urban et al's [^urban] installed base data:
+
+| Device             | Installed Base     |
+| ------------------ | ------------------ |
+| Desktop            | 37%                |
+| Laptop             | 63%                |
+| Monitor            | 52%                |
+
+When the end user’s computer cannot be determined between a laptop (with or without a monitor) and a desktop, we can use the installed base data to estimate that a personal computer consumes an average of 53.2W (rounded up):
+
+$$
+(0.37 \times 72.3) + (0.63 \times 17.1) + (0.52 \times 30) = 53.12W
+$$
+
+With these values, it is possible to estimate the power consumption by users when using a product or service on a device. The more information available about what devices are used and how much time users spend on these products, the more accurate the estimates can be.
+
+To calculate energy use (Wh), the power (W) of the device is divided by the time factor (hours = 1, minutes = 60, seconds = 3600), then multiplied by the duration. For example, assume a blog page with an estimated reading time of 5 minutes. The energy consumption of a user on a laptop (without a monitor) can be estimated using the following calculation:
+
+$$ Energy\ (Wh) = 17.1W / 60 \times 5 = 1.4Wh $$
```

Comment:
I'd suggest "17.1W \times 5 / 60" but it's not critical.

Reply:
I agree it looks better but I don't know if the text above would also want to change. I find it a little confusing, what we're really trying to say is that power is multiplied by the time in hours, so if you're dealing with minutes you need to convert.